dashboard/server/api/server.js

```diff
@@ -0,0 +1,21 @@
+const controllers = require('../../lib/controllers/run');
```

Comment:
Should be `runController`

```suggestion
const runController = require('../../lib/controllers/run');
```